catalog-rest-service/src/main/resources/ui/src/axiosAPIs/topicsAPI.ts

```diff
@@ -16,6 +16,7 @@
 */
 
 import { AxiosResponse } from 'axios';
+import { Topic } from 'Models';
```

Comment:
Import this from Topics under `generated` folder

Reply:
For this PR we can use from models and we can change it in another Separate PR.